Jenkinsfile

```diff
@@ -27,6 +27,10 @@ try {
             sh "CCACHE_DIR=${CCACHE_DIR} VERSION=${VERSION} build/run make -j${parallel} release"
         }
 
+        stage('Style Check') {
+            sh "CCACHE_DIR=${CCACHE_DIR} VERSION=${VERSION} build/run make -j${parallel} lint"
+        }
+
```

Comment:
It doesnt look like lint is being run as part of build and install. https://github.com/rook/rook/blob/master/build/makelib/golang.mk#L132

I dont want to do that yet since this is more of an experimental check. There are a lot of errors that we need to fix in our source code to make lint check pass. Until then, it is probably better not to make it as part of build/install target